front/package.json

```diff
@@ -165,8 +165,11 @@
     "start": "import-meta-env-prepare -x .env.example && vite",
     "build": "vite build",
     "test": "import-meta-env-prepare -x .env.example && vitest",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx src",
     "test-coverage": "import-meta-env-prepare -x .env.example && vitest run --coverage",
-    "generate-types": "npx @rtk-query/codegen-openapi src/config/openapi-middleware-config.ts && npx @rtk-query/codegen-openapi src/config/openapi-editoast-config.ts",
+    "generate-types": "yarn generate-types-main && yarn generate-types-eslint",
+    "generate-types-main": "npx @rtk-query/codegen-openapi src/config/openapi-middleware-config.ts && npx @rtk-query/codegen-openapi src/config/openapi-editoast-config.ts",
+    "generate-types-eslint": "eslint --fix src/common/api/osrdMiddlewareApi.ts src/common/api/osrdMiddlewareApi.ts",
```

Comment:
can you merge these 3 tasks into one?

Reply:
I wanted to avoid having so much shit crammed in one single

Reply:
please use a post script then: https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts
you could also move it all into a shell script, and replace it all with `exec generate-types.sh`

Reply:
done ! I like the idea of using of script, much cleaner.